mappings/net/minecraft/entity/ai/brain/WalkTarget.mapping

```diff
@@ -0,0 +1,5 @@
+CLASS any net/minecraft/entity/ai/brain/WalkTarget
```

Comment:
What does this class do? Seems odd for a WalkTarget to hold a LookTarget

Reply:
It's what get stored in the `walk_target` memory module.